### ISSUE TITLE
Erroneous warning about VCGT being non-linear when creating 3D LUT

### ISSUE THREAD

**Issue by ledoge** [eoyilmaz/displaycal-py3#182]
When generating a 3D LUT using the 3D LUT tab (e.g. in .cube format) with "Apply calibration (vcgt)" checked, something weird happens: After creating the 3D LUT, clicking `Save 3D LUT...` results in [this](https://github.com/eoyilmaz/displaycal-py3/blob/3b25f05f63d27e4a9e6e197773dbcc3506124c80/DisplayCAL/wxLUT3DFrame.py#L605) code running. To determine whether it should show a warning about the VCGT currently being non-linear, it iterates over all displays and checks if the VCGT is linear. This part makes sense. However, this warning is also shown when you only have a single display connected (`real_displays_count == 1`), even if its VCGT is linear.

I'm curious if anyone has an idea why this would be in the code, as I can't come up with an explanation for it. I did find [the commit by Florian](https://sourceforge.net/p/dispcalgui/code/3775/) that added the check, but it doesn't explain why.

For reference, the warning text reads:

>The display device's video card gamma tables 1D LUT calibration is non-linear, but the 3D LUT contains applied 1D LUT calibration. Make sure to manually reset the video card gamma tables to linear before using the 3D LUT, or create a 3D LUT without calibration applied (to do the latter, enable “Show advanced options” in the “Options” menu and disable “Apply calibration (vcgt)” as well as “Create 3D LUT after profiling” in the 3D LUT settings, then create a new 3D LUT).

**Comment by eoyilmaz** [comment]
You are right, I'm not sure why the ``real_displays_count`` is in that condition at all, and unfortunately Florian didn't include a comment on why ``real_displays_count == 1`` should be a condition. I believe it makes sense when the logical operation was not ``or`` but ``and`` meaning there is a real display and the gamma is erroneously applied both in VCGT and in 3D LUT while there is not a problem to apply the VCGT to the 3D LUT if the display is virtual (then ``real_displays_count > 0`` is more correct), what do you think about that?
